js/reference.js

```diff
@@ -1516,6 +1516,30 @@ var ERMrest = (function(module) {
         },
 
         /**
+         * This function takes in a tuple and generates a reference that is
+         * constrained based on the domain_filter_pattern annotation. If this
+         * annotation doesn't exist, it returns this (reference)
+         * `this` is the same as column.reference
+         * @param {ERMrest.ReferenceColumn} column - column that `this` is based on
+         * @param {Object} data - tuple data with potential constraints
+         * @returns {ERMrest.Reference} the constrained reference
+         */
+        filteredRef: function(column, data) {
```

Comment:
`filteredRef` should be a property of [`ReferenceColumn`](https://github.com/informatics-isi-edu/ermrestjs/blob/master/js/reference.js#L2579) not `Reference`.